recipes/speex/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "1.2rc2" %}
```

Comment:
Normally we prefer to use stable release as opposed to `rc` or `dev` releases. Given there is a `1.2.0` release of `speex`, is there some reason not to use it?